### DISCUSSION TITLE
How to add webp image in templete

### DISCUSSION THREAD

**Discussion by ludimassa** [Oksydan/falcon#333] [Q&A]
Hi everyone if I try to add image in the _dev/img folder webpack is unable to compile all the files
How can I add some image like background and no have problems?

**Comment by Oksydan** [comment]
Hi @ludimassa,

Yes, it looks like there is a "problem" with the webpack test. TBH I didn't think anyone would want to use the webp file. You can modify this test `/{ png|jpg|gif|svg)$/` and just add a new file extension `webp`.
https://github.com/Oksydan/falcon/blob/master/_dev/webpack/webpack.parts.js#L104-L123
